src/proto/business/client.go

```diff
@@ -56,12 +57,25 @@ func (h *Handler) CreateClient(ctx context.Context, grpcApiAddress string, token
 
 }
 
+const serverNamePrefix = "zbusinessapi"
+
+func verifyPeerServerName(p *advancedtls.VerificationFuncParams) (*advancedtls.VerificationResults, error) {
+	if p.Leaf == nil {
+		return nil, fmt.Errorf("missing leaf certificate")
+	}
+	for _, name := range p.Leaf.DNSNames {
```

Comment:
I am unsure how this works so I trust you :D

Reply:
Thanks :D. It just basically compares prefixes of the domains inside a certificate and not the exact match.